Lesson19/Ece/index.html

```diff
@@ -1,25 +1,35 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<title>DOM Exercise</title>
-</head>
-<body>
-  <main id="main">
-    <h1>My Website</h1>
-    <img src="https://picsum.photos/800/400" alt="Image 1">
-    <p id="first-paragraph">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean consectetur tellus non elit molestie, quis ornare purus imperdiet. Nam sodales sapien eu ultrices lacinia.</p>
-    <img src="https://picsum.photos/800/400?random=2" alt="Image 2">
-    <ul>
-      <li>List Item 1</li>
-      <li>List Item 2</li>
-      <li>List Item 3</li>
-    </ul>
-    <p id="paragraph1">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ratione, quibusdam molestiae. Soluta pariatur minus, reiciendis consectetur nobis accusantium quia natus molestiae voluptatum non doloribus dolorum dolor voluptates at! Labore, nulla.
-    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Illum esse repudiandae veniam ullam neque numquam, nam laudantium eveniet! Iure assumenda porro officia magni reiciendis temporibus quis consequatur fugit libero! Et.
-    </p>
-  </main>
-  <script src="script.js"></script>
-</body>
+  <head>
+    <title>DOM Exercise</title>
+  </head>
+  <body>
+    <main id="main">
+      <h1>My Website</h1>
+      <img id = "img1" src="https://picsum.photos/800/400" alt="Image 1" />
+      <p id="first-paragraph">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean
+        consectetur tellus non elit molestie, quis ornare purus imperdiet. Nam
+        sodales sapien eu ultrices lacinia.
+      </p>
+      <img src="https://picsum.photos/800/400?random=2" alt="Image 2" />
+      <ul>
```

Comment:
You try to select this element by id but this element does not have an id. To fix it, apply the suggestion.
```suggestion
      <ul id="ul">
```